doc/credit.rst

```diff
@@ -0,0 +1,394 @@
+
+:orphan:
```

Comment:
This file is autogenerated, better to review it by looking at the rendered HTML and code that generates it.